Dockerfile.build-tools

```diff
@@ -112,6 +112,40 @@ RUN for package in Capture::Tiny DateTime Devel::Cover Digest::MD5 File::Spec JS
     && make install \
     && rm -rf ../lcov.tar.gz
 
+# Compile and install the static OpenSSL library
+ENV OPENSSL_VERSION=3.2.2
+ENV OPENSSL_PREFIX=/usr/local/openssl
```

Comment:
I've used different variable here to not to conflict with system variable i.e. `OPENSSL_PREFIX`.

Reply:
ignore this comment, it was conflicting only with Makefile.